scripts-available/CDB_Overviews.sql

```diff
@@ -21,6 +21,17 @@ END;
 $$ LANGUAGE PLPGSQL VOLATILE;
 
 
+-- Return whether overviews are disabled by configuration.
+-- Access to CDB_Conf is wrapped by this function to enable
+-- execution by regular users.
+CREATE OR REPLACE FUNCTION _CDB_OverviewsDisabled()
+RETURNS BOOLEAN
+AS $$
+BEGIN
+  RETURN COALESCE(CDB_Conf_GetConf('disable_overviews')::text, 'false') = 'true';
+END;
+$$ LANGUAGE PLPGSQL STABLE SECURITY DEFINER;
```

Comment:
I presume that you tested without security definer and it failed..

Reply:
Also, did you test with IMMUTABLE option for SECURITY DEFINER (Instead of STABLE)??
If IMMUTABLE works, it is best since this will be exposed to all in windshaft API.. 
If not, then STABLE is good..

Reply:
Agreed. That is the is the definition of "idempotent". They should have used that word instead!

Reply:
@tylerparsons Could you explictly layout who you expect the owner of this function will be?

I'd expect `postgres`. Then, you're basically creating a public read-only wrapper to CDB_Conf values regarding `disable_overviews`. That would mean that any user can run `_CDB_OverviewsDisabled()`, but none can run `CDB_Conf_GetConf`. Is that right?

Thanks in advance for the clarification.

Reply:
@gfiorav correct - I tested with `CDB_Conf_GetConf`, direct access to `CDB_Conf` and the `_CDB_OverviewsDisabled` function - the latter is the only one that can be accessed by regular users.

Reply:
I'm not sure how this privilege gets assigned, or who is considered the "owner" in each case. I trust that for our case, this is thoroughly tested. In any case, it's not a huge breach otherways.

Thanks for the clarification!